apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -503,20 +503,24 @@ impl Display for LicenseEnforcementReport {
 
 /// License controls availability of certain features of the Router. It must be constructed from a base64 encoded JWT
 /// This API experimental and is subject to change outside of semver.
-#[derive(Debug, Clone, Default)]
+#[derive(Debug, Clone, Default, Eq, PartialEq)]
```

Comment:
We are doing this because the plugin will need to use the claims and send them somewhere, is that right? There's no additional state explosion that we can do initially?

Reply:
talked in slack; updated! new struct, tighter contract, better derives